### PR TITLE
feat: support telegram login

### PR DESCRIPTION
**Pull request by Ehco1996**
as title

### CODE REVIEW

**Comment by Ehco1996** [comment]
@Calcium-Ion ptal

**Comment by Calcium-Ion** [comment]
感谢pr，稍后处理

File: makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FRONTEND_DIR = ./web
+BACKEND_DIR = .
+
+.PHONY: all build-frontend start-backend
+
+all: start-frontend start-backend
+
+build-frontend:
```
Comment (suclogger) [inline makefile:8]: Seems `start-frontend` is meant to be build-frontend.

Comment (Ehco1996) [reply makefile:8]: addressed in https://github.com/Calcium-Ion/new-api/pull/79/commits/699fe256d0d85b9b7bae6dd6483fd9a2e793a579

**Pull request merged**